data/utils.py

```diff
@@ -0,0 +1,36 @@
+"""Utils for data"""
+
+from typing import List
+
+
+def build_holder_relation(symbols: List[str], api_call, fields, threshold=1):
+    """Parses all holder information (institutional, mutual) to construct a relation matrix"""
+    companies = list(
+        filter(
+            None,
+            [api_call(symbol) for symbol in fields],
+        )
+    )
+
+    companies_sorted_and_stripped = [
+        sorted(holders, key=lambda holder: int(holder["shares"]), reverse=True)[:10]
```

Comment:
this 10 here really is a magic number. we should think about extracting it in a meaningfull variable name or passing it to the function as a "hyperparameter"